ocean_lib/data_provider/data_service_provider.py

```diff
@@ -350,6 +351,14 @@ def get_url(config):
     def get_api_version():
         return DataServiceProvider._remove_slash(os.getenv(ENV_PROVIDER_API_VERSION, DataServiceProvider.API_VERSION))
 
+    @staticmethod
+    def get_service_endpoints():
+        if DataServiceProvider.provider_info is None:
+            config = ConfigProvider.get_config()
+            DataServiceProvider.provider_info = requests.get(config.provider_url)
+            DataServiceProvider.provider_info = DataServiceProvider.provider_info.json()
+        return DataServiceProvider.provider_info
```

Comment:
can this return `DataServiceProvider.provider_info['serviceEndpoints']`?

Reply:
ok, sure.